src/components/ScholarshipList.jsx

```diff
@@ -39,8 +40,14 @@ const useStyles = makeStyles((theme) => ({
 
 function ScholarshipList({ scholarships }) {
   const classes = useStyles();
+  const [shareDialogOpen, setShareDialogOpen] = useState(false);
+  const closeShareDialog = () => setShareDialogOpen(false);
+
+  const [shareSiteLink, setShareSiteLink] = React.useState('');
+  const [shareSiteTitle, setShareSiteTitle] = React.useState('');
+
   const shareFn = (id, data) => () => {
-    const title = `${data.amount} - ${data.name} | ${BRAND_NAME}`;
+    const title = `https://${data.amount} - ${data.name} | ${BRAND_NAME}`;
     const url = `${window.location.hostname}/scholarships/${id}`;
```

Comment:
You added the `https://` in the wrong place.
```suggestion
    const title = `${data.amount} - ${data.name} | ${BRAND_NAME}`;
    const url = `https://${window.location.hostname}/scholarships/${id}`;
```

Reply:
Ok I did not notice this, good catch